packages/pods-core/src/v2/podConfig/GoogleDocsConfig.ts

```diff
@@ -71,7 +73,7 @@ export function createRunnableGoogleDocsV2PodConfigSchema(): JSONSchemaType<Runn
       exportScope: {
         type: "string",
       },
-      description: {
```

Comment:
nit: can we leave description? We haven't been using this, but it let's someone add some text that will show up in the quick pick: 

<img width="503" alt="image" src="https://user-images.githubusercontent.com/3203268/182983275-dc7d1535-32d0-4181-bb5e-fa6e83cdb12c.png">

Reply:
`description` is still there in every export pod config, it is just removed from `Runnable` config of google docs export(not a required property to run gdoc export)

Reply:
Ah, makes sense. Misread the code